chirpstack/chirpstack-concentratord/files/gateway-id.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+SLOTS="|slot1|slot2"
+OUTPUT=""
+
+oldIFS=$IFS
+IFS="|"
+
+for SLOT in $SLOTS; do
+    
+    URL="/tmp/concentratord_command"
+    if [ "$SLOT" != "" ]; then
+        URL="/tmp/concentratord_${SLOT}_command"
+    fi
+
+    if [ -e "$URL" ]; then
+        GATEWAY_ID=$( /usr/bin/gateway-id -c "ipc://$URL" )
+        if [ $? -eq 0 ]; then
+            [ "$OUTPUT" != "" ] && OUTPUT="$OUTPUT / "
+            OUTPUT="${OUTPUT}${GATEWAY_ID}"
+            [ "$SLOT" != "" ] && OUTPUT="${OUTPUT} ($SLOT)"
+        fi
+    fi
+
+done
+
+IFS=$oldIFS
+
+[ "$OUTPUT" = "" ] && OUTPUT="could not read gateway_id"
```

Comment:
We might want to change `gateway_id` into `gateway id` in this case :)

Reply:
You mean in the message? "Could not read Gateway ID"?

Reply:
Correct